modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -124,10 +124,18 @@ Double the space if the server is an ISS Master.
 
 |===
 
-
-
 endif::[]
 
+ifeval::[{suma-content} == true]
+[WARNING]
+====
+Maximum number of clients which can be managed by a single {productname} server is 10 000.
```

Comment:
I am not sure about the wording here as was pointed by other colleagues. We should be going for like "It is advisable or strongly recommended to move to Hub setup ad involve consultancy once you have more than 10k number of minions" , not exactly what I mentioned but along those lines.